zulipterminal/model.py

```diff
@@ -179,24 +179,40 @@ def get_message_ids_in_current_narrow(self) -> Set[int]:
             current_ids = self.index['all_starred']
         return current_ids.copy()
 
-    @asynch
-    def _start_presence_updates(self) -> None:
+    def _start_presence_initially(self) -> None:
         """
-        Notify server of user's presence each minute (version 1a)
+        Notify server of user's presence initially.
         """
-        # FIXME: version 1b: Also use 'response' to update user list
-        # FIXME: Version 2: call endpoint with ping_only=True only when
-        #        needed, and rely on presence events to update
-        while True:
-            response = self.client.update_presence(
+        self._notify_server_of_presence()
+        self._start_presence_updates()
+
+    def _notify_server_of_presence(self) -> None:
```

Comment:
I like that you've extracted this method, but perhaps extract it first with a refactor commit, then add features?

Reply:
Will do so :+1: